qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -81,37 +80,6 @@ public void launchPatientDashboardPage() {
 		assertNotNull(dashboardPage.getActiveVisitList());
 	}
 
```

Comment:
For the sake of uniformity I aligned the steps definition file to match the feature file steps.